k8s/upgrades/0.8.0-0.8.1/patch-strategy-recreate.json

```diff
@@ -0,0 +1,4 @@
+[
```

Comment:
why we need this?

Reply:
patch-strategy-recreate.json is used to patch the strategy from `rollingUpdate` to `Recreate`